### PR TITLE
SONARJAVA-967: Fix FP on S2230

### PR DESCRIPTION
**Pull request by m-g-sonar**

### CODE REVIEW

**Comment by SonarTech** [comment]
SonarQube analysis reported no new issues.

File: java-checks/src/main/java/org/sonar/java/checks/TransactionalMethodVisibilityCheck.java [hunk 1]
```diff
@@ -52,7 +52,7 @@
   @Override
   public void visitNode(Tree tree) {
     MethodTree method = (MethodTree) tree;
-    boolean isPublic = method.modifiers().modifiers().contains(Modifier.PUBLIC);
+    boolean isPublic = method.modifiers().modifiers().contains(Modifier.PUBLIC) || method.symbol().owner().isInterface();
```
Comment (benzonico) [inline java-checks/src/main/java/org/sonar/java/checks/TransactionalMethodVisibilityCheck.java]: Why not check the flag directly on symbol then ?

**Comment by SonarTech** [comment]
SonarQube analysis reported no new issues.

**Comment by benzonico** [comment]
LGTM

**Pull request closed without merging**